app/src/main/java/com/cortado/minionify/accumulators/IAccumulator.java

```diff
@@ -0,0 +1,7 @@
+package com.cortado.minionify.accumulators;
+
+public interface IAccumulator {
```

Comment:
we usually dont put `I` in interfaces. you dont have to rename, it's just a convention that isnt used in java.

Reply:
Done, I rather be on Java's conventions.